pom.xml

```diff
@@ -18,10 +18,10 @@
     <ramlfiles_path>${basedir}/ramls</ramlfiles_path>
     <ramlfiles_util_path>${ramlfiles_path}/raml-util</ramlfiles_util_path>
     <ramlfiles_acq_models_path>${ramlfiles_path}/acq-models</ramlfiles_acq_models_path>
-    <raml-module-builder.version>27.1.1</raml-module-builder.version>
+    <raml-module-builder.version>28.1.0</raml-module-builder.version>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <jmockit.version>1.48</jmockit.version>
-    <argLine />
+<!--    <argLine />-->
```

Comment:
Why is it commented? I thought we needed it. If it is not added then on building locally it fails with Surefire Exception.